firecrown/ccl/sources/sources.py

```diff
@@ -7,7 +7,7 @@
 from ..systematics import IdentityFunctionMOR, TopHatSelectionFunction
 
 
-__all__ = ['WLSource', 'NumberCountsSource']
+__all__ = ['WLSource', 'NumberCountsSource', 'ClusterSource']
```

Comment:
Do all of these refer to clusters? I'm a bit unclear on what each is.

Reply:
Nope. These match the ccl tracers. Only the last one is for clusters.